src/api/route-services/pipeline-response.js

```diff
@@ -14,12 +14,7 @@ const experimentService = new ExperimentService();
 
 const getPipelineStatus = require('../general-services/pipeline-status');
 
-const embeddingWorkRequest = require('../../utils/hooks/embeddingWorkRequest');
-const clusteringWorkRequest = require('../../utils/hooks/clusteringWorkRequest');
-
 const pipelineHook = new PipelineHook();
-pipelineHook.register('configureEmbedding', embeddingWorkRequest);
```

Comment:
do we still need to have this `hookRunner` class at all? And do we still need to run it on line 109?

Reply:
No, that is why it is deleted now.

Reply:
but line 109 when you call it is not deleted. And you are still initialising it in line 20

Reply:
Fair enough. Removed!